fineract-provider/src/main/java/org/apache/fineract/portfolio/loanproduct/api/LoanProductsApiResourceSwagger.java

```diff
@@ -1326,15 +1326,15 @@ private PutLoanProductsProductIdRequest() {}
         @Schema(example = "false")
         public Boolean useBorrowerCycle;
         @Schema(example = "2")
-        public Integer repaymentFrequencyType;
+        public Long repaymentFrequencyType;
```

Comment:
This looks wrong to me hence this field is the ordinal of an enum.

Reply:
Done.